docs/usage/rsync/database_example.rst

```diff
@@ -2,64 +2,106 @@
 Rsync Database Example
 ======================
 
-The following example will use the Rsync replication method and take a Snapshot
-at the destination. A MySQL database will be used as the example application.
+The following example will use the Rsync replication method to periodically
+replicate a MySQL database.
 
-First, create the destination and deploy the ReplicationDestination
-configuration.
+First, create the destination Namespace and deploy the ReplicationDestination
```

Comment:
Great changes overall!  Like the tabs a lot in the install section.

Particularly with this rsync section (although maybe this applies to rclone and restic as well) - do we need some extra explanation about scenarios with multiple clusters?  I.e. which steps need to be done on which cluster?  I realize this does add complexity - so maybe we could just add something at the top level saying for multi-cluster scenarios that the replicationdestination needs to be created on the dest cluster and replicationsource on source.

The area I know that we got a lot of criticism is related - i.e. how to copy the secret around (particularly cluster-to-cluster).  The ACM docs are trying to clarify this a bit, and the example there specifically will show a cluster-to-cluster example, so perhaps we can get away with not explaining that part here?

Reply:
I could add a page under rsync about how to copy a Secret from cluster-to-cluster. Then we could link it when we discuss that step in the examples.

Reply:
That sounds great 👍

Reply:
I've added explanations (and links to) for SSH key copying as well as the trade-off in choosing the type of Service to use: https://volsync--136.org.readthedocs.build/en/136/usage/rsync/index.html#rsync-specific-considerations